atomic-reactor.spec

```diff
@@ -86,6 +69,23 @@ BuildRequires:  python3-six
 BuildRequires:  python3-osbs >= 0.48
 BuildRequires:  python3-responses
 %endif # with_check
+%else
+BuildRequires:  python2-devel
+BuildRequires:  python-setuptools
+%if 0%{?with_check}
+BuildRequires:  pytest
+BuildRequires:  python-dockerfile-parse >= 0.0.11
+%if (0%{?fedora} >= 29)
+BuildRequires:  python2-docker
+%else
+BuildRequires:  python-docker-py
```

Comment:
This else block can probably go away: Fedora < 29 has reached EoL anyway (there is anothe occurrence of this case above which should also be removed).

Reply:
ok